src/metatile_behavior.c

```diff
@@ -1400,3 +1405,31 @@ bool8 MetatileBehavior_IsTrainerHillTimer(u8 metatileBehavior)
     else
         return FALSE;
 }
+
+bool32 MetatileBehavior_IsSignpost(u32 mb)
+{
+#if OW_AUTO_SIGNPOST == TRUE
```

Comment:
is there a reason those and the following are prerprocs?

Reply:
It was a holdover from when I was going to keep the tileset changes.

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/5044/commits/31fc49734a1770d4c27f10efebb58069a15fb23b